src/Saas.Permissions/Saas.Permissions.Api/Controllers/PermissionsController.cs

```diff
@@ -1,11 +1,15 @@
-﻿using Saas.Permissions.Api.Exceptions;
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.Identity.Web.Resource;
+using Saas.Permissions.Api.Exceptions;
 using Saas.Permissions.Api.Interfaces;
 using System.Net;
 
 namespace Saas.Permissions.Api.Controllers;
 
 [Route("api/[controller]")]
 [ApiController]
+[Authorize]
+[RequiredScope("access_as_user_via_admin_api")]
```

Comment:
It might be better to split the scopes to permission_read & permission_write
Consider the situation where a reporting client would need to read permissions, but we don't want to be able to make changes.  I know we've been saying only admin should access these, but really the requirement is only applications with proper business logic who are trusted should be accessing this.